src/fsharp/FSharp.Core/array2.fs

```diff
@@ -95,20 +93,20 @@ namespace Microsoft.FSharp.Collections
             let count2 = length2 array 
             let b1 = base1 array 
             let b2 = base2 array 
-            for i = b1 to b1+count1 - 1 do 
-              for j = b2 to b2+count2 - 1 do 
+            for i = b1 to b1 + count1 - 1 do 
+              for j = b2 to b2 + count2 - 1 do 
                 action array.[i,j]
 
         [<CompiledName("IterateIndexed")>]
         let iteri (action : int -> int -> 'T -> unit) (array:'T[,]) =
             checkNonNull "array" array
-            let count1 = length1 array 
-            let count2 = length2 array 
-            let b1 = base1 array 
-            let b2 = base2 array 
+            let count1 = length1 array
+            let count2 = length2 array
+            let b1 = base1 array
+            let b2 = base2 array
             let f = OptimizedClosures.FSharpFunc<_,_,_,_>.Adapt(action)
-            for i = b1 to b1+count1 - 1 do 
-              for j = b2 to b2+count2 - 1 do 
+            for i = b1 to b1+count1 - 1 do
```

Comment:
since it's cleanup spaces around the +   ?

Reply:
done